torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1560,7 +1610,7 @@ def relation_with_digit(expr, op, digit):
                 t = transform(s)
                 if t == s:
                     continue
-                left, op, right = t.split(" ")
+                left, op, right = t.split(" ", 2)
```

Comment:
🤔

Reply:
lol yeah we expect `symbol == expr` here, so 2 blanks. Will add comment.

Reply:
Using re split now.